service/src/main/java/uk/gov/hmcts/reform/fpl/service/DraftCMOService.java

```diff
@@ -224,9 +376,8 @@ private void prePopulateHearingDateSelection(List<Element<HearingBooking>> heari
     }
 
     private String getRespondentFullName(RespondentParty respondentParty) {
-        String firstName = defaultIfNull(respondentParty.getFirstName(), "");
-        String lastName = defaultIfNull(respondentParty.getLastName(), "");
-
+        String firstName = defaultString(respondentParty.getFirstName());
+        String lastName = defaultString(respondentParty.getLastName());
         return String.format("%s %s", firstName, lastName);
```

Comment:
Easier you added `buildFullName` to `Party` class. Use it here to remove duplicate.